starter-code/css/main.css

```diff
@@ -6,3 +6,59 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* ----- reset ----- */
+
+/* ----- Main Settings ----- */
+
+/* html {
+    box-sizing: border-box;   
+}   *, *:before, *:after {     box-sizing: inherit;   } */
+
+body {
+    font-style: montserrat, sans-serif;
+}
+
+/* ----- Header ----- */
+
+.spotifyLogo {
+    max-height: 200px;
+    max-width: 200px;
+    padding: 10px;
+    float: left;
+}
+
+.navigationBar {
+    float: right;
+    vertical-align: middle;
+    padding: 14px 30px;
+    word-spacing: 20px;
+}
+
+ul li {
+    display: inline;
+}
+
+header:after {
+    content: '';
+    display: block;
+    clear: both;
+}
+
+
+/* ----- Corpus ----- */
+
+.section {
+    background-image: url("/Users/santiagoramos/Ironhack/Week1/lab-css-spotify-clone/starter-code/images/landing.jpg") 
```

Comment:
USE RELATIVE PATHS!!!!!!!!!!! (With the path you are giving, you are the only one who can see the images). It should look like this:
```
background-image: url("../images/landing.jpg")
```